config/etfuturum/world.cfg

```diff
@@ -0,0 +1,192 @@
+    I:deepslateGenerationMode=0
+
+    # The dimensions the deepslate layer (deepslate generation mode 0) should not spawn in. Does nothing if other deepslate generation modes are used.
+    I:deepslateLayerDimensionBlacklist <
+        -1
+        1
+     >
+
+    # Treat the deepslate layer dimension blacklist as a whitelist instead, so it will ONLY generate in those dimensions, instead of excluding those dimensions from generation. [default: false]
+    B:deepslateLayerDimensionBlacklistAsWhitelist=false
+
+    # How high up deepslate and tuff goes. If deepslateGenerationMode is 0, all stone up to this y level (with a scattering effect a few blocks before then) are replaced with deepslate. If it's 1, the patches can generate to that Y level. [range: 0 ~ 255, default: 22]
+    I:deepslateMaxY=22
+
+    # Whether or not Deepslate will overwrite dirt (Only works when deepslate generation mode is set to 0) [default: true]
+    B:deepslateReplacesDirt=true
+
+    # Whether or not Deepslate will overwrite granite, diorite, andesite (Only works when deepslate generation mode is set to 0) [default: true]
+    B:deepslateReplacesStones=true
+
+    # Can ancient debris generate next to air? [default: false]
+    B:enableAirDebris=false
+
+    #  [default: true]
+    B:enableAmethystGeodes=true
+
+    # Replaces coarse dirt in biomes it (dirt:1) generates in such as shattered savannas or mesa plateaus. [default: true]
+    B:enableCoarseDirtReplacement=true
+
+    # Generate 20 more veins of gold ore from Y 32 to Y 80 in any Mesa biome. [default: true]
+    B:enableExtraMesaGold=true
+
+    #  [default: true]
+    B:enableFossils=true
+
+    # Generates extra mineshafts in mesa biomes up to y80. If fences are enabled, dark oak wood is used. [default: true]
+    B:enableMesaMineshaft=true
+
+    # Note: Ocean monuments currently do not have guardians [default: true]
+    B:enableOceanMonuments=true
+
+    # Use a namespaced ID, + optionally meta (max 3) to choose the block that makes up fossils. The max meta is 3 because the rotations will change the meta. North/South is the meta + 4 and East/West is + 8.
+    # Netherlicious bone block is "netherlicious:BoneBlock" and UpToDate bone block is "uptodate:bone_block".
+    # If the chosen block does not exist then fossils will not generate. If Netherlicious is installed, its bone block will be used if this is set to "etfutrum:bone_block" and Et Futurum Requiem bone blocks are disabled. [default: etfuturum:bone]
+    S:fossilBlockID=etfuturum:bone
+
+    # The dimension IDs of the dimensions the fossil structures should not spawn in. Fossils will also not spawn in any dimension that is not an instance of WorldProviderSurface
+    I:fossilDimensionBlacklist <
+     >
+
+    # Treat the fossil dimension blacklist as a whitelist instead, so fossils will ONLY generate in those dimensions, instead of excluding those dimensions from generation. [default: false]
+    B:fossilDimensionBlacklistAsWhitelist=false
```

Comment:
whitelist please

Reply:
Done. And set the overworld and the nether as the two allowed dimensions.